### DISCUSSION TITLE
Add Ctrl+S to save/close in O2M dialog

### DISCUSSION THREAD

**Discussion by AaronBeaudoin** [directus/directus#16214] [Feature Requests]
In the detail page, you can use Ctrl+S to "Save" the item. This is a great way to save clicks. However, when I open an item in an O2M field item list, it opens the item in a dialog on the right, inside of which Ctrl+S doesn't do anything. It would be much more intuitive if Ctrl+S closed the dialog, equivalent to pressing the purple "Save" button in the top right. That way if I'm nested into several O2M fields I can just spam Ctrl+S to go all the way up and save the item from the "root" detail page.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
